docs/oas/business.yaml

```diff
@@ -2439,6 +2439,109 @@ paths:
                 type: object
                 properties: {}
               examples:
+                incorporation-application-success-response:
+                  summary: Incorporation Application Response  
+                  value:
+                    errors:
+                      - null
+                    filing:
+                      business:
+                        identifier: Tqcbbbc157  
+                      header:
+                        accountId: 6164
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: API Specs
+                        colinIds: []
+                        comments: []
+                        date: '2024-07-25T16:17:57.191108+00:00'
```

Comment:
date format should be `'yyyy-mm-dd'` here 
![image](https://github.com/user-attachments/assets/18d3b759-ddbb-4fb1-b9cf-b4644e8dd5bf)

Reply:
Still working on the spec